schema-loader/src/main/java/com/scalar/db/schemaloader/util/either/Either.java

```diff
@@ -0,0 +1,3 @@
+package com.scalar.db.schemaloader.util.either;
+
+public abstract class Either<A, B> {}
```

Comment:
Maybe you can refer to this implementation of `Either`:
https://github.com/google/guava/issues/1149

I think it should have factory methods, and `isLeft()` and `isRight()` so that we can make the code cleaner. Using `instanceof` and a cast is basically not good.

Reply:
Thanks for the reference. I changed them in [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/219473b5710f1f7154ae5075eaa1531d48b854d6)